mix.exs

```diff
@@ -38,11 +38,14 @@ defmodule MLLP.MixProject do
       {:ranch, "~> 1.8.0"},
       {:elixir_hl7, "~> 0.6.0"},
       {:backoff, "~> 1.1.6"},
+      {:gen_state_machine, "~> 3.0"},
       {:ex_doc, "~> 0.24.2", only: :dev, runtime: false},
       {:dialyxir, "~> 1.1.0", only: [:dev, :test], runtime: false},
       {:mix_test_watch, "~> 1.0.2", only: :dev, runtime: false},
       {:mox, "~> 1.0.0", only: :test},
-      {:excoveralls, "~> 0.14.4", only: :test, runtime: false}
+      {:excoveralls, "~> 0.14.4", only: :test, runtime: false},
+      ## Debugging
+      {:replbug, github: "bokner/replbug"}
```

Comment:
We're going to need to make this optional. Especially because it pulls in from git repo.